src/test/java/com/browserstack/SingleTest.java

```diff
@@ -10,12 +10,22 @@ public class SingleTest extends BrowserStackTestNGTest {
 
     @Test
     public void test() throws Exception {
-        driver.get("https://www.google.com/ncr");
-        WebElement element = driver.findElement(By.name("q"));
-        element.sendKeys("BrowserStack");
-        element.submit();
-        Thread.sleep(5000);
-
-        Assert.assertTrue(driver.getTitle().matches("(?i)BrowserStack - Google Search"));
+    	// navigate to bstackdemo
+        driver.get("https://www.bstackdemo.com");
+        
+        // Check the title
+        Assert.assertTrue(driver.getTitle().matches("StackDemo"));
+        
+        // Save the text of the product for later verify
+        String productOnScreenText = driver.findElement(By.xpath("//*[@id=\"1\"]/p")).getText();
```

Comment:
Do we need to change scripts inside https://github.com/browserstack/testng-browserstack/blob/master/src/test/java/com/browserstack/suite/ too?